.vscode/extensions.json

```diff
@@ -0,0 +1,10 @@
+{
+    "recommendations": [
+        "ms-vscode.cpptools-extension-pack",
```

Comment:
J'ajouterais aussi :
- Kconfig for the Zephyr Project
- DeviceTree for the Zephyr Project

Reply:
oui, je suis rendu là dans mes recherches. À venir prochainement.